run.sh

```diff
@@ -8,4 +8,4 @@ else
   export LLAMA_N_CTX=8182
 fi
 
-poetry run python -m uvicorn skynet.main:app --reload
+poetry run python -m skynet.main
```

Comment:
Why the change?

Reply:
to mirror what we are running the deployment, currently it was bypassing the normal startup flow from the `main` module

Reply:
aha!